### PR TITLE
Add docs link and Optimize language display in UI

### PR DESCRIPTION
**Pull request by xiaojun207**
1、MainLayout add docs entry link；
2、Optimize language display and selection；
3、"docs.zincsearch.io" -> "docs.zincsearch.com"

### CODE REVIEW

**Comment by xiaojun207** [comment]
> 1、MainLayout add docs entry link； 2、Optimize language display and selection； 3、"docs.zincsearch.io" -> "docs.zincsearch.com"

The previous submission overwritten

**Comment by hengfeiyang** [comment]
image

There is an error in my test.

**Comment by xiaojun207** [comment]
> image
> 
> There is an error in my test.

Sorry, let me check again

**Comment by hengfeiyang** [comment]
We think the header is too heavy, We should delete the text of `documentation` and `language`, it will be slightly.

And i think you can use `padding="xs sm"` replace `padding="xs lg"`

Thank you.

File: web/src/layouts/MainLayout.vue [hunk 1]
```diff
@@ -14,9 +14,35 @@
...
+            :label="t('menu.documentation')"
+            href="https://docs.zincsearch.com"
+            target="_blank"
+          />
+        </div>
+
+        <div>
+          <q-btn-dropdown unelevated no-caps padding="xs sm">
+            <template #label>
+              <div class="row no-wrap" style="margin-right: -12px;">{{ selectedLanguage.label }}</div>
```
Comment (hengfeiyang) [inline web/src/layouts/MainLayout.vue:30]: I think use css define better than fixed style.

like:

```
<style lang="scss">
.q-header .q-btn-dropdown__arrow {
  margin-left: 0;
}
</style>
```

**Review by hengfeiyang — APPROVED** [review]

**Pull request merged**